### PR TITLE
DOC-7114: Updated screenshots for monitoring with the UI

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/manage/pages/monitor/ui-monitoring-statistics.adoc [hunk 1]
```diff
@@ -270,7 +268,7 @@ The displayed statistics are:
 | The amount of swap space in use.
 
 | free RAM
-| The amount of RAM available.
+| The amount ozf RAM available.
```
Comment (amarantha-k) [inline modules/manage/pages/monitor/ui-monitoring-statistics.adoc]: typo

File: modules/manage/pages/monitor/ui-monitoring-statistics.adoc [hunk 2]
```diff
@@ -640,103 +563,79 @@ The statistics are as follows:
...
 
-| CAS badval per sec.
-| Number of CAS operations per second using an incorrect CAS ID for data that this bucket contains.
+| deletes per sec.
+| Delete operations per second as a result of the incoming XDCR data stream.
 
-| CAS misses per sec.
-| Number of CAS operations per second for data that this bucket does _not_ contain.
+| total ops per sec.
+| Total of all the operations per second.
```
Comment (amarantha-k) [inline modules/manage/pages/monitor/ui-monitoring-statistics.adoc]: "of all operations per second"?

**Review by amarantha-k — APPROVED** [review]
Two minor comments to be fixed before merging.

File: modules/manage/pages/monitor/ui-monitoring-statistics.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/manage/pages/monitor/ui-monitoring-statistics.adoc]: Thanks, fixed.

File: modules/manage/pages/monitor/ui-monitoring-statistics.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/manage/pages/monitor/ui-monitoring-statistics.adoc]: Fixed.

**Pull request merged**